toucan_connectors/snowflake_common.py

```diff
@@ -98,36 +101,36 @@ def _execute_query_internal(
         execution_start = timer()
         cursor = connection.cursor(DictCursor)
         query_res = cursor.execute(query, query_parameters)
-        execution_end = timer()
-        execution_time = execution_end - execution_start
+
+        query_generation_time = timer() - execution_start
         self.logger.info(
-            f'[benchmark][snowflake] - execute {execution_time} seconds',
+            f'[benchmark][snowflake] - execute {query_generation_time} seconds',
             extra={
                 'benchmark': {
                     'operation': 'execute',
-                    'execution_time': execution_time,
+                    'query_generation_time': query_generation_time,
                     'connector': 'snowflake',
                     'query': query,
                 }
             },
         )
-        self.set_execution_time(execution_time)
+        self.set_query_generation_time(query_generation_time)
```

Comment:
Not necessarily in the PR but: what is the interest of having setters for these properties? It seems exactly the same than `self.query_generation_time = ` to me

Reply:
True, i just changed the names, because it was already there..
and am not sure, but... i think... 🤔 we don't know if this function get used by laputa... if we are only on the context of this PR... and some of the setters are used by other methods as callbacks... see [line 187](https://github.com/ToucanToco/toucan-connectors/blob/2e853eb1869db8746816ca54dda028c5ccb6d8e5/toucan_connectors/snowflake_common.py#L187)

The code will be adapt in consequence on laputa.

Reply:
Yes, it's not related to this PR anyway ^^